skills/relay-merge/references/operator-emergencies.md

```
@@ -0,0 +1,46 @@
+
+These are operator-only emergency paths for relay-merge. The standard merge path lives in `skills/relay-merge/SKILL.md` § 1.
+
+Emergency escape hatch:
+
+```bash
+node ${CLAUDE_SKILL_DIR}/scripts/finalize-run.js --repo . --run-id "$RUN_ID" --skip-review "hotfix" --json
+```
+
+`finalize-run.js --skip-review` bypasses reviewer invocation, so `model_hints.review` is a non-consumer on that path.
+
+#### Operator-only force finalize for non-ready runs
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h4

(MD001, heading-increment)